### PR TITLE
Added Notification menu (top right) to link to SeqWare Run Report

### PR DESCRIPTION
**Pull request by slazicoicr**

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -158,6 +161,15 @@ server <- function(session, input, output) {
       current.run(createAppDT(all.runs.dt()[name == input$run, path]))
       all.studies <- current.run()$Study
       updateSelectInput(session, "study", choices = all.studies)
+      
+      # Add links that lead to useful places
+      output$notificationMenu <- renderMenu({
+        dropdownMenu(
+          notificationItem("SeqWare Run Report", status = "info", icon = icon("copy"), href = generateRunReportURL(input$run)),
+          notificationItem("MISO Run Report", status = "info", icon = icon("cog"), href = generateMisoRunURL(input$run)),
```
Comment (callunity) [inline app.R]: Are you strongly attached to the "copy" and "cog" icons? Many web pages use icons very similar to these to indicate "copy" and "settings", which is not what they're being used for here. What do you think about something like [extrnal-link-alt](https://fontawesome.com/icons/external-link-alt?style=solid) for both of them?

Comment (slazicoicr) [reply app.R]: For unknown reasons, that icon does not display. What about below:
image

Comment (callunity) [reply app.R]: It [looks like it should work if you specify](https://shiny.rstudio.com/reference/shiny/1.0.1/icon.html) `icon = icon("external-link-alt", lib = "font-awesome")` but I have no strong preference for the external link icon over the link icon you've got here.
Could the "Dropdown menu" link icon be changed to be the same icon as the notification item link icons?

Comment (slazicoicr) [reply app.R]: Still did not work, so I am following link icon strategy. Looks like this:
image

**Review by callunity — APPROVED** [review]

**Pull request merged**